tasks/section1.yml

```diff
@@ -428,7 +428,7 @@
       - rule_1.1.9
 
 - name: "SCORED | 1.1.10 | AUDIT | Ensure noexec option set on /var/tmp partition"
-  command: /bin/true
+  shell: mount | grep /var/tmp | grep noexec
   register: result
```

Comment:
Audit tasks should register verbose variable name that ends with `_audit`. As you are adding AUDIT checks please change `register: result` to `register: `*`descriptive_check_name_audit`*